### DISCUSSION TITLE
Cannot pair Bosch "Plug compact EU"

### DISCUSSION THREAD

**Discussion by JWomser** [Koenkk/zigbee2mqtt#20209] [Devices]
Maybe someone has an idea or a hint for me.

I bought a Bosch plug which should be supported (https://www.zigbee2mqtt.io/devices/BSP-FZ2.html) but I am not able to pair it to Z2M.

To pair it, I scanned the the device code with my smart phone and got 95 characters starting with "RB01SG0D83101826480030000000000000000000B43522FFFE367317DLK" and followed by 36 hexadecimal digits. This 36 hexadecimal digits are printed as "Install code" at the device too.

I reset the plug to fabric defaults and then I used the Z2M-dialog "Settings -> Tools -> Add install code" with the complete 95 character code but got the error "Add install code is not supported".

I tried only the printed "Install code" (the last 36 hex chars) but I got the error "Unsupported install code, got 36 chars, expected 95 or 91".

I am using HA 2023.12.2 and Z2M 1.34.0-1 with a "Conbee II" coordinator.

**Comment by TTr0n** [comment]
Any solutions to this ?
Same problem here ...

**Comment by superpower10** [comment]
Same here
